src/main/java/org/generationcp/middleware/domain/dms/PhenotypicType.java

```diff
@@ -28,14 +28,16 @@ public enum PhenotypicType {
 			"SITENO"), "TRIAL ENVIRONMENT"), //
 
 	GERMPLASM(Arrays.asList(TermId.GERMPLASM_ENTRY_STORAGE.getId(), TermId.ENTRY_NUMBER_STORAGE.getId(), TermId.ENTRY_GID_STORAGE.getId(),
-			TermId.ENTRY_DESIGNATION_STORAGE.getId(), TermId.ENTRY_CODE_STORAGE.getId()), Arrays.asList("ENTRY", "ENTRY_NO", "ENTRYNO"),
-			"GERMPLASM ENTRY"), //
+			TermId.ENTRY_DESIGNATION_STORAGE.getId(), TermId.ENTRY_CODE_STORAGE.getId()), Arrays.asList(""), "GERMPLASM ENTRY"), //
```

Comment:
@cuenyad why Arrays.asList("") is needed?

Reply:
Hi  @clarysabel, these are the places used in the code. We could discuss it tomorrow.

https://github.com/IntegratedBreedingPlatform/Middleware/blob/edd069520cbc992ef190cabffdc4a472c6846a96/src/main/java/org/generationcp/middleware/operation/parser/WorkbookParser.java#L100
https://github.com/IntegratedBreedingPlatform/Middleware/blob/76d5703ed1283389f72ed316569d17158499eecf/src/main/java/org/generationcp/middleware/domain/etl/Workbook.java#L420